shapes.py

```diff
@@ -0,0 +1,29 @@
+class Shape:
+    pass
+
+class TShape:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.squares = [(x,y),(x+1,y),(x+2,y),(x+1,y+1)]
```

Comment:
Maybe we can use a `Unit` class array instead of tuples.

```python
class Unit:
    def __init__(self, x, y):
        self.x = x
        self.y = y
```